packages/spark-core/settings/_settings.scss

```diff
@@ -495,7 +509,7 @@ $masthead-border-bottom: $masthead-border-bottom-size solid $gray !default;
 $masthead-breakpoint: 54rem !default;
 $masthead-column-width: 89rem !default;
 $big-nav-column-width: 64rem !default;
-$big-nav-bg: $gray-tint-25 !default;
+$big-nav-bg: $gray !default;
```

Comment:
Make sure you update the masthead page Sass Table to reflect the new default big nav bg color.